lib/segment/src/index/field_index/geo_index/mod.rs

```diff
@@ -49,28 +52,43 @@ impl GeoMapIndex {
         }
     }
 
+    pub fn new_mmap(path: &Path) -> OperationResult<Self> {
+        Ok(GeoMapIndex::Mmap(Box::new(MmapGeoMapIndex::load(path)?)))
+    }
+
     pub fn builder(db: Arc<RwLock<DB>>, field: &str) -> GeoMapIndexBuilder {
         GeoMapIndexBuilder(Self::new(db, field, true))
     }
 
-    fn db_wrapper(&self) -> &DatabaseColumnScheduledDeleteWrapper {
-        match self {
-            GeoMapIndex::Mutable(index) => index.db_wrapper(),
-            GeoMapIndex::Immutable(index) => index.db_wrapper(),
+    #[cfg(test)]
+    pub fn builder_immutable(db: Arc<RwLock<DB>>, field: &str) -> GeoMapImmutableIndexBuilder {
+        GeoMapImmutableIndexBuilder {
+            index: Self::new(db.clone(), field, true),
+            field: field.to_owned(),
+            db,
+        }
+    }
+
+    pub fn mmap_builder(path: &Path) -> GeoMapIndexMmapBuilder {
+        GeoMapIndexMmapBuilder {
+            path: path.to_owned(),
+            dynamic_index: DynamicGeoMapIndex::new(),
         }
     }
 
     fn points_count(&self) -> usize {
         match self {
-            GeoMapIndex::Mutable(index) => index.points_count,
-            GeoMapIndex::Immutable(index) => index.points_count,
+            GeoMapIndex::Mutable(index) => index.get_points_count(),
+            GeoMapIndex::Immutable(index) => index.get_points_count(),
+            GeoMapIndex::Mmap(index) => index.get_indexed_points(),
```

Comment:
The method name is different. Is this intended?

Reply:
Fixed